### ISSUE TITLE
Missing XACK command

### ISSUE THREAD

**Issue by grubeli** [dragonflydb/dragonfly#1869]
I'm using Dapr in my microservices and have recently switched from Redis to Dragonfly as the broker for the pubsub and store components.
After the switch, I'm encountering an error with the XACK command not being recognized by Dragonfly. This command is used in my setup to remove processed messages from the pending entries list in a stream consumer group. Without it, my services are receiving the same request repeatedly as the request is not being removed after handling.
I've checked the Dragonfly documentation but couldn't find a clear answer.
Error provided by Dapr logger:
```bash
ERRO[6814] Error acknowledging Redis message 1694931309634-0: ERR unknown command `XACK`  app_id=static component="pubsub-xxx (pubsub.redis/v1)" instance=analysis scope=dapr.contrib type=log ver=1.11.3
```
Xack is used for Redis Streams - which Dapr uses for PubSub. Maybe dragonfly DB doesn't implement any RedisStreams features.

Thank you.

**Comment by romange** [comment]
We currently do not support all the STREAMS commands.

**Comment by chakaz** [comment]
I'm changing the labels of this issue from `bug` to `enhancement` for easier internal tracking.

**Comment by theyueli** [comment]
hi @grubeli, this has now been supported. Would you like to test your use case again to see if this works for you? Thanks!

**Comment by EvertonSA** [comment]
i would like to test, but i cant find a version available, does 1.10 contains this?

**Comment by theyueli** [comment]
> i would like to test, but i cant find a version available, does 1.10 contains this?

you could try our weekly built container  image via 

docker pull http://ghcr.io/dragonflydb/dragonfly-weekly
